src/lib/wizards/functions/cover.svelte

```diff
@@ -106,12 +106,15 @@
                                 </li>
                             {/each}
                         {:then response}
-                            {@const runtimes = new Map(response.runtimes.map((r) => [r.$id, r]))}
-                            {@const templates = quickStart.runtimes.filter((_template) =>
-                                runtimes.has(_template.name)
-                            )}
+                            {@const templates = quickStart.runtimes.filter((_template) => {
+                                return response.runtimes.some((runtime) =>
+                                    runtime.$id.includes(_template.name)
+                                );
+                            })}
```

Comment:
This looks like it reverted a previous change. We should make sure to use better data types when available to avoid doing n^2 operations (iterating over an array in an array).

Reply:
But now the template name is only going to include the language used not the version so `has` will not work anymore :thinking: